tests/e2e-playwright/README.md

```diff
@@ -0,0 +1,14 @@
+Auto generate new test
+`playwright codegen sim4life.py`
+
+Run test locally with headed mode
+`pytest -s test sim4life.py --tracing on --headed `
+
+Check test results output
+`playwright show-trace test-results/tests-sim4life-py-test-billable-sim4life-chromium/trace.zip`
+
+Run debug mode
+`PWDEBUG=1 pytest --s tests/sim4life.py`
+
+Run test in different browsers
+`pytest -s tests/sim4life.py --tracing on --html=report.html --browser chromium --browser firefox`
```

Comment:
Just an addition on how to debug traces on non development machines

```suggestion
`pytest -s tests/sim4life.py --tracing on --html=report.html --browser chromium --browser firefox`


# Viewing the traces locally on a machine with no simcore

- make sure you have `npm` installed in your system
- make sure npx is installed `npm install -g npx`
- finally debug the track `npx playwright show-trace path-to/trace.zip`
```

Reply:
npx is not necessary (probably you were checking node documentation) python is enough